_rules/table-header-cells-refer-to-data-cells-d0f69e.md

```diff
@@ -0,0 +1,258 @@
+    </tr>
+  </tbody>
+</table>
+```
+
+#### Passed Example 3
+
+The `columnheader` has corresponding `cells` within the same `table`, by usage of `headers` attribute.
+
+```html
+<table>
+  <tr> 
+    <td headers="header1">
+      Projects
+    </td>
+    <th id="header1">
+      Projects
+    </th> 
+  </tr>
+</table>
+```
+
+#### Passed Example 4
+
+The `columnheader` has corresponding `cells` within the same `table`, by usage of `aria-labelledby` attribute.
+
+```html
+<table>
+	<tr>
+		<td aria-labelledby="hd2">Time</td>
+		<th id="hd2">05:44</th>
+	</tr>
+</table>
+```
+
+#### Passed example 5
+
+Each `th` element has corresponding `cells` within the same `table`, by usage of `scope` attribute.
+
+```html
+<table>
+  <caption>Hogwarts Houses stars</caption>
+  <tr>
+      <th scope="col">Name</th>
+      <th scope="col">House</th>
+  </tr>
+  <tr>
+      <th scope="row">Harry Potter</th>
+      <td>Gryffindor</td>
+  </tr>
+  <tr>
+      <th scope="row">Cedric Diggory</th>
+      <td>Hufflepuff</td>
+  </tr>
+  <tr>
+      <th scope="row">Cho Chang</th>
+      <td>Ravenclaw</td>
+  </tr>
+    <tr>
+      <th scope="row">Severus Snape</th>
+      <td>Slytherin</td>
+  </tr>
+</table>
+```
+
+### Failed
+
+#### Failed Example 1
+
+There are no corresponding `cells` for each `th` element within the `table`.
+
+```html
+<table>
+	<tr>
+		<th scope="col">Name</th>
+		<th scope="col">Age</th>
+		<th scope="col">Birthday</th>
+	</tr>
+	<tr>
+		<th scope="row">Jackie</th>
+		<td>5</td>
+		<td></td>
+	</tr>
+	<tr>
+		<th scope="row">Beth</th>
+		<td>8</td>
+	</tr>
+</table>
+```
+
+#### Failed Example 2
+
+`th` has corresponding empty `cell`.
+
+```html
+<table>
+  <tr> 
+    <th>Time</th> 
+  </tr>
+  <tr> 
+    <td></td> 
+  </tr>
+</table>
+```
+
+#### Failed Example 3
+
+No data `cells` exist within the `table`.
+
+```html
+<table>
+	<tr>
+		<td>axe</td>
+		<td role="columnheader">AXE</td>
+	</tr>
+</table>
+
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+The rule only applies only to `table > th` element. The `table` has no `th` elements.
+
+```html
+<table>
+  <tr>
+    <td>John</td>
+    <td>Doe</td>
+  </tr>
+  <tr>
+    <td>Jane</td>
+    <td>Doe</td>
+  </tr>
+</table>
+```
+
+#### Inapplicable Example 2
+
+The rule only applies only to `table` & not layout table.
+
+```html
+<div role="table">
+ <div role="row">
+  <div class="columnheader">Projects</div>
+  <div class="columnheader">Exams</div>
+ </div>
+ <div role="row">
+  <div role="cell">15%</div>
+  <div role="cell">15%</div>
+ </div>
+</div>
+```
+
+#### Inapplicable Example 3
+
+The rule only applies to `table` element that is included in the accessibility tree. The `table` is marked as `role=presentation`.
+
+```html
+<table role="presentation">
+  <tr> 
+    <th>Time</th> 
+  </tr>
+  <tr> 
+    <td>12:00</td> 
+  </tr>
+</table>
+```
+
+#### Inapplicable Example 4
+
+The rule only applies to `table` element that is included in the accessibility tree.
```

Comment:
Both this and the previous examples address the same applicability issue. It would be more interesting to have one with a table that is not visible to cover all applicability issues.